docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
-
-If you're using a bundler such as [Webpack](https://webpack.js.org/), use the `import` syntax to import all components. To initialise them, use the `initAll` function:
-
-```JS
-import { initAll } from 'govuk-frontend'
+If your `image` and `font` folders are in different folders, set the `$govuk-images-path` and `$govuk-fonts-path` variables to your folders. For example:
 
-initAll()
+```SCSS
+$govuk-images-path: "/{YOUR-IMAGES-FOLDER}/";
+$govuk-fonts-path: "/{YOUR-FONTS-FOLDER}/";
+@import "govuk-frontend/govuk/all";
 ```
 
-If you're using a bundler such as [Browserify](http://browserify.org/), you may need to use the CommonJS `require`:
+You can also use your own function to generate paths, for example if you're using `asset-pipeline` in [sass-rails](https://github.com/rails/sass-rails). Set the `$govuk-image-url-function` variable to the name of your function.
 
-```JS
-const GOVUKFrontend = require('govuk-frontend')
+### 4. Add JavaScript and initialise components
 
-GOVUKFrontend.initAll()
-```
+You can either:
 
-#### Import individual components
+- add GOV.UK Frontend's JavaScript file to your HTML
+- import the JavaScript using a bundler like [Webpack](https://webpack.js.org/)
 
-If you're using a bundler such as Webpack, use the `import` syntax to import a component:
+You must also initialise GOV.UK Frontend's components by calling the `initAll` function. If you do not initialise, some components will not work correctly for users who have accessible needs or use older browsers.
 
-```JS
-import { Radios } from 'govuk-frontend'
-```
+### Add the JavaScript file to your HTML
 
-If you're using a bundler such as [Browserify](http://browserify.org/), you may need to use the CommonJS `require`:
+First either:
 
+- route requests to the `node_modules/govuk-frontend/govuk/all.js` file
+- copy the file into your project folder
 
-```JS
-const GOVUKFrontend = require('govuk-frontend')
+Then add GOV.UK Frontend's Javascript file before the closing `</body>` tag of your HTML page or page template, and run the `initAll` function to initialise all the components.
 
-const Radios = GOVUKFrontend.Radios
+```html
+<body>
+...
+  <script src="node_modules/govuk-frontend/govuk/all.js"></script>
+  <script>
+    window.GOVUKFrontend.initAll()
+  </script>
+</body>
 ```
 
-GOV.UK Frontend components with JavaScript behaviour have the `data-module` attribute set in their markup.
+You can also select specific components, then call `initAll` with a `scope` parameter that includes only those components. For example, to initialise only [radios](https://design-system.service.gov.uk/components/radios/):
 
-You can use this attribute to initialise the component manually, this may be useful if you are adding markup to a page after it has loaded.
-
-To initialise the first radio component on a page, use:
-
-```js
-var $radio = document.querySelector('[data-module="govuk-radios"]')
-if ($radio) {
-  new Radios($radio).init()
-}
+```html
+...
```

Comment:
This section confuses the scope feature with individual component initialisation, they should be separately explained as they solve different issues and are different features.

I don't think this example would work as given.

Scope is used to determine a part of the page to look for components to initialise, for example if you've added some new markup that needs to be initialised.

Reply:
Ah yes, sorry - it seems like I've merged two different sections into one somehow. I'll rework.